app/views/welcome/index.html.haml

```diff
@@ -1,44 +1,52 @@
-.root-landing-page
-  %header.root-landing-banner-header
+%main.root-landing-page
+  %header.jumbotron.root-landing-banner-header.bg-image-bike-hero
     .container
-      .row
-        .root-landing-header-text
-          %p
-            %strong
-              = number_with_delimiter Counts.recoveries
-            bikes recovered.
-          %h1
-            <span>Bike</span> <span>Registration</span> <span>that</span> <span>Works</span>
-          %a.btn.btn-lg.btn-success.root-action-btn{ href: new_user_path }
-            Get started, it's free
-        .root-landing-signup
-          - unless current_user.present?
-            = form_for User.new do |f|
-              .form-group
-                = f.label :name, 'full name', class: 'sr-only'
-                = f.text_field :name, placeholder: 'name', class: 'form-control'
-              .row
-                .form-group.col-md-6
-                  = f.label :email, class: 'sr-only'
-                  = f.email_field :email, placeholder: 'email', class: 'form-control'
-                .form-group.col-md-6
-                  = f.label :password, class: 'sr-only'
-                  = f.password_field :password, placeholder: 'password', class: 'form-control'
+      .root-landing-header-text
+        %h2
+          Over
+          = number_with_delimiter(@recovery_count)
+          Bikes Recovered
+        %h1.upcase
+          Bike registration that works
+        %h2
+          = link_to new_organization_path do
+            Find out how &raquo;
```

Comment:
This is actually targeting consumers (not organizations).

Currently, it should scroll to "BUT HOW DOES IT WORK?" on the homepage itself.